package.json

```diff
@@ -33,7 +37,8 @@
     "grunt-jscs-checker": "~0.4.2",
     "grunt-sass": "^0.18.1",
     "load-grunt-tasks": "^0.4.0",
-    "pretty": "^0.1.2"
+    "pretty": "^0.1.2",
+    "foundation-sites": "zurb/foundation@5.5.*"
```

Comment:
This doesn't seem to allow the package to be installed on Travis or my local machine.

I'm going to change this to `"foundation-sites": "~5.5.2"` unless I hear from you that this is a really bad idea :)